README.md

```diff
@@ -66,18 +66,42 @@ Contains:
 
 2. Filter to subscriptions
 
-<img src="https://i.gyazo.com/eccd333b76e6ac8b948d9b012bd62301.png" width="640">
+![Humble Filters](resources/Humble_filters.png)
 
 3. Group by subscriptions
 
-<img src="https://i.gyazo.com/869901d9dc6d730d4744ea11eb7c0b8f.png" width="640">
+![Humble Group By](resources/Humble_Groupby.png)
 
 3. Bookmark
 
-<img src="https://i.gyazo.com/d3cb0876788849a9117f739e3b5d875a.png" width="400">
+![Humble bookmarked](resources/Humble_bookmarked.png)
 
 4. Rename a new bookmark with right-click
 
+## FAQ
+
+### Can I group my Humble games and Choice games altogether?
+
+You have to create a new filtered Humble Bundle bookmark, proceeding as follows:
+
+1. Click on the defauit _Humble Bundle_ bookmark: you see only the owned games in this view, and your goals is to have a similar view that lists both owned and subscritptions games.
+
+2. Change the view filter selecting the funnel icon, then the Status menu -> and then flagging both _Owned_ and _Subcriptions_
+
+![HumbleBundle_Filters](resources/HumbleBundle_Filters.png)
+
+3. Bookmark the search query you just composed clicking on the small bookmark flag
+
+![Humble bookmarked](resources/Bookmarking.png)
+
+3. The view that you've just created will show in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Owned + Choices_.
```

Comment:
```suggestion
3. The view that you've just created will appear in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so, click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Owned + Choices_.
```

looks more natural for me (but again: I'm not an Englishman, fell free to ignore if you know better)